### DISCUSSION TITLE
Update laravel/prompts dependency to 0.3.1 (latest)

### DISCUSSION THREAD

**Discussion by ElvisIsKing666** [lunarstorm/laravel-ddd#72] [Ideas]
Hi - thanks for your work. I just updated both my laravel project to laravel 11 and my lunarstorm/laravel-ddd to the latest and was forced to downgrade laravel/prompts to 0.1.25. I'm not sure what work it would take to upgrade to the latest version but others using laravel 11 will be grateful.

**Comment by jaspertey** [comment]
Thanks for letting me know, I will look into this asap.

**Reply by jaspertey** [reply]
@ElvisIsKing666 - Should be resolved in the latest release, could you confirm on your end and let me know?

**Reply by ElvisIsKing666** [reply]
Yup, good to go - muchas gracias
